src/xdocs/config_javadoc.xml

```diff
@@ -782,7 +807,22 @@ public int checkReturnTag(final int aTagIndex,
   &lt;property name=&quot;allowMissingParamTags&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Example: </p>
+        <source>
+/**
+ * @param param1 The first number
+ */
+private int addTwoNumbers(int param1, int param2){  // ok, param tags not necessary
```

Comment:
Please keep java code identical except for trailing comments that mark violation or explain why ok.
Java code is same, Config is changing , trailing comments are changing.

If you have problems with line length, please keep name of method and parameters as foo, p1, p2,....
It is better to put training comment on same line that actual checkstyle is placing (see CLI out put)